### PR TITLE
Separated robot claim from robot start route

### PR DESCRIPTION
**Pull request by csu**
Split off robot claim into its own route (formerly was combined with robot start). I have yet to test this on the robot.

### CODE REVIEW

**Comment by csu** [comment]
Also, this will probably create a merge conflict with #6

File: src/robot_start_stop.py [hunk 1]
```diff
@@ -15,12 +16,17 @@ def claim_and_start_robot():
     except CalledProcessError:
         return jsonify({'status': 'error', 'message': 'Failed to claim robot.'})
 
+    return jsonify({'status': 'success', 'message': 'Robot claimed.'})
+
+@blueprint.route('/start', methods=['POST'])
+def start_robot():
```
Comment (jstnhuang) [inline src/robot_start_stop.py:23]: Can you add @login_required here and for all API routes?

Comment (csu) [inline src/robot_start_stop.py:23]: I did that in #6. Should I just do it all in this pull request?

Comment (jstnhuang) [inline src/robot_start_stop.py:23]: Oops, I didn't realize that. Sorry. I just merged the other one. Can you go ahead and try to resolve the merge conflict? Thanks!

**Comment by csu** [comment]
Merge conflict resolved.

File: src/robot_start_stop.py (hunk 1 above)
Comment (jstnhuang) [inline src/robot_start_stop.py:23]: Authentication still missing from here :)

Comment (csu) [inline src/robot_start_stop.py:23]: Should be fixed now

Comment (jstnhuang) [inline src/robot_start_stop.py:23]: Are you sure? It still looks like it's missing an @login_required annotation before start_robot()

Comment (csu) [inline src/robot_start_stop.py:23]: Whoops, should be pushed now.

**Pull request merged**